crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/signed_subintent.rs

```diff
@@ -31,6 +31,10 @@ impl SignedSubintent {
         )
         .expect("Compiling after initialization is always valid")
     }
+
+    pub fn decompiled(compiled: Vec<u8>) -> Result<Self> {
```

Comment:
hmm do we use this verb tense for ctor? maybe we do in a few places. But I would prefer `decompling` I think